From f9e2d5c67571714b37f17526a53ef979bd4af7bf Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Fri, 4 Dec 2015 09:28:38 -0500
Subject: [PATCH 186/260] 01_users: Handle GRUB_PASSWORD better.

Only handle GRUB_PASSWORD not GRUB2_PASSWORD (as that's more likely to
be acceptable upstream).

Related: rhbz#1284370

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 util/grub-setpassword.8 | 2 +-
 util/grub.d/01_users.in | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/util/grub-setpassword.8 b/util/grub-setpassword.8
index 5973abef4..49200a848 100644
--- a/util/grub-setpassword.8
+++ b/util/grub-setpassword.8
@@ -9,7 +9,7 @@
 \fBgrub-setpassword\fR outputs the user.cfg file which contains the hashed GRUB bootloader password. This utility only supports configurations where there is a single root user.
 
 The file has the format:
-GRUB_2PASSWORD=<\fIhashed password\fR>.
+GRUB2_PASSWORD=<\fIhashed password\fR>.
 
 .SH OPTIONS
 .TP
diff --git a/util/grub.d/01_users.in b/util/grub.d/01_users.in
index facd409e7..db2f44bfb 100644
--- a/util/grub.d/01_users.in
+++ b/util/grub.d/01_users.in
@@ -2,7 +2,7 @@
 cat << EOF
 if [ -f \${prefix}/user.cfg ]; then
   source \${prefix}/user.cfg
-  if [ -n \${GRUB2_PASSWORD} ]; then
+  if [ -n "\${GRUB2_PASSWORD}" ]; then
     set superusers="root"
     export superusers
     password_pbkdf2 root \${GRUB2_PASSWORD}
-- 
2.13.0

